doc/smart-answers/erb-templates.md

```diff
@@ -1,27 +1,30 @@
 # ERB templates
 
-Content is defined in `render_content_for` blocks and with the name of the intent of the content:
+Content is defined using one of three helpers to indicate the format:
 
 ```erb
-<% render_content_for :title do %>
-  Some amazing title
+<% text_for :title do %>
+  This is plain text, any HTML characters used will be espaced
 <% end %>
 ```
 
-The type of content can be explicitly set with the `format:` option:
+```erb
+<% govspeak_for :body do %>
+  This uses the GOV.UK markdown dialect, govspeak, and will be converted to
+  HTML.
+<% end %>
+```
 
 ```erb
-<% render_content_for :title, format: :html do %>
-  <h1>Some amazing title</h1>
+<% html_for :body do %>
+  <p>This is HTML, used when we need fine grained control over content</p>
 <% end %>
 ```
 
-We support Govspeak (`:govspeak`), plain text (`:text`) and HTML (`:html`). See docs for template pages for the default formats of different content blocks.
+For examples of which of these to use in which contexts see relevant documentation for different contexts:
 
 * [Landing page templates](erb-templates/landing-page-template.md)
 * [Question templates](erb-templates/question-templates.md)
 * [Outcome templates](erb-templates/outcome-templates.md)
 
-We remove all leading spaces from the content in the `render_content_for` blocks for Govspeak and Text formats. This allows us to indent the content in the `render_content_for` blocks without having to worry about it affecting the generated HTML when it's processed using Govspeak.
-
```

Comment:
Do we convey the point that indentation is removed else where in the documentation? I might have missed it

Reply:
I feel this is sufficiently implied through the examples. I didn't think this added a huge amount of value in this context. Do you think it's something people benefit from? could add something to individual functions.

Reply:
Sorry defo not a blocker for this PR. I just anecdotally remember someone else finding this useful https://github.com/alphagov/smart-answers/pull/4502#discussion_r420162488

Reply:
Argh sorry - I didn't realise it had been added so recently. I can understand that it raised an eyebrow for me to delete it.

Reply:
I've added some info into the examples to say indentation is removed.